model/src/lib.rs

```diff
@@ -329,13 +332,76 @@ impl TokenPair {
     }
 }
 
+#[derive(Copy, Eq, PartialEq, Clone, Default)]
+pub struct DomainSeparator(pub [u8; 32]);
+
+// For command line argument parsing.
```

Comment:
Where do we plant o use this?

Reply:
ups, yeah I will remove the comment. But the fromstr trait is very handy. It is for example used here:
```
let expected_domain_separator: DomainSeparator =
            "9d7e07ef92761aa9453ae5ff25083a2b19764131b15295d3c7e89f1f1b8c67d9"
                .parse()
                .unwrap();
```
in the tests.